wiki/Gameplay/Judgement/Notelock/es.md

```diff
@@ -0,0 +1,35 @@
+---
+tags: 
+  - note lock
+  - jittering notes
+  - shaking notes
+  - notas bloqueadas
+  - notas temblorosas
+  - agitando notas
+---
+
+# Notelock
+
+![](img/notelock.gif "An example of notelock: the second note is inaccessible")
```

Comment:
alt-texts and tooltips of images need to be translated

for reference:

`![alt-text](image_link "tootlip")`

Reply:
ok, its solved